app/models/contestant.rb

```diff
@@ -1,3 +1,9 @@
 class Contestant < ApplicationRecord
   validates_presence_of :name, :age, :hometown, :years_of_experience
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects 
+
+  def project
+    projects.order(:name).pluck(:name).join(", ")
```

Comment:
Nice work encapsulating this in a model method. Just be sure to add a model test for any instance or class methods in the model!